tools/tokenserver/loadtests/locustfile.py

```diff
@@ -60,15 +76,18 @@ def test_oauth_success(self):
 
     @task(100)
     def test_invalid_oauth(self):
-        token = self._make_oauth_token(status=400)
+        token = self._make_oauth_token(
+            self.email,
+            key=INVALID_OAUTH_PRIVATE_KEY
```

Comment:
To create an invalid token, we sign the JWT with a private key that doesn't correspond with the public key set on Tokenserver. In this case, we just generate a new private key with every run of the load tests.